src/App.js

```diff
@@ -97,8 +97,8 @@ const App = ({ locale, location, dependencies: { appSessionRef, sessionManager }
           <PrivateRoute path="/reports/" exact requireSiteTracking component={Reports} />
           <PrivateRoute path="/integrations/shopify" exact component={Shopify} />
           <PrivateRoute path="/reports/master-subscriber" exact component={MasterSubscriber} />
-          <PrivateRoute path="/reports/subscriber-history" exact component={SubscriberHistory} />
-          <PrivateRoute path="/reports/subscriber-gdpr" exact component={SubscriberGdpr} />
+          <PrivateRoute path="/subscribers/:email/:section" component={Subscribers} />
+          <PrivateRoute path="/reports/:oldParam" exact component={SubscribersRedirect} />
```

Comment:
```suggestion
          {/* This is to keep backward compatibility with /reports/subscriber-history and /reports/subscriber-history */}
          <PrivateRoute path="/reports/:oldParam" exact component={SubscribersRedirect} />
```

Reply:
What do you think about a TODO comment to delete this after is changed in MasterSubscribers?

Reply:
Mmm, I would keep it for more time, what happens if a user bookmarked it?

Reply:
In that case we should keep this change forever 🤔